### PR TITLE
SystweakSoftware.RightBackup version Right Backup

### PR DESCRIPTION
**Pull request by systweaksoftware**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19611)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Missing Version?
@denelon how did this get past validation?

**Comment by denelon** [comment]
@OfficialEsco, it passed due to a match between the folder structure and the version. I agree this does not look like a package version we want in the repository (assuming "Right Backup" is not a proper version for this package).

File: manifests/s/SystweakSoftware/RightBackup/Right Backup/SystweakSoftware.RightBackup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SystweakSoftware.RightBackup
+PackageVersion: Right Backup
```
Comment (OfficialEsco) [inline manifests/s/SystweakSoftware/RightBackup/Right Backup/SystweakSoftware.RightBackup.installer.yaml:4]:
```suggestion
PackageVersion: 2.1.1000.6914
```

File: manifests/s/SystweakSoftware/RightBackup/Right Backup/SystweakSoftware.RightBackup.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SystweakSoftware.RightBackup
+PackageVersion: Right Backup
```
Comment (OfficialEsco) [inline manifests/s/SystweakSoftware/RightBackup/Right Backup/SystweakSoftware.RightBackup.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 2.1.1000.6914
```

File: manifests/s/SystweakSoftware/RightBackup/Right Backup/SystweakSoftware.RightBackup.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: SystweakSoftware.RightBackup
+PackageVersion: Right Backup
```
Comment (OfficialEsco) [inline manifests/s/SystweakSoftware/RightBackup/Right Backup/SystweakSoftware.RightBackup.yaml:4]:
```suggestion
PackageVersion: 2.1.1000.6914
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
I somehow didn't see the PackageVersion because i expected some kind of number

image
image

**Pull request closed without merging**